packages/flutter_driver/test/src/real_tests/extension_test.dart

```diff
@@ -543,6 +543,73 @@ void main() {
     expect(await getOffset(OffsetType.center), const Offset(40 + (100 / 2), 30 + (120 / 2)));
   });
 
+  testWidgets('getText', (WidgetTester tester) async {
+    await silenceDriverLogger(() async {
+      final FlutterDriverExtension extension = FlutterDriverExtension((String arg) async => '', true);
+
+      Future<String> getTextInternal(SerializableFinder search) async {
+        final Map<String, String> arguments = GetText(search, timeout: const Duration(seconds: 1)).serialize();
+        final Map<String, dynamic> result = await extension.call(arguments);
+        if (result['isError'] as bool) {
+          return null;
+        }
+        return GetTextResult.fromJson(result['response'] as Map<String, dynamic>).text;
+      }
+
+      await tester.pumpWidget(
+          MaterialApp(
+              home: Scaffold(body:Column(
+                key: const ValueKey<String>('column'),
+                children: <Widget>[
+                  const Text('Hello1', key: const ValueKey<String>('text1')),
+                  Container(
+                      height:25.0,
+                      child: RichText(
+                          key: const ValueKey<String>('text2'),
+                          text: const TextSpan(text: 'Hello2'))
+                  ),
+                  Container(
+                      height:25.0,
+                      child: EditableText(
+                          key: const ValueKey<String>('text3'),
+                          controller: TextEditingController(text: 'Hello3'),
+                          focusNode: FocusNode(),
+                          style: const TextStyle(),
+                          cursorColor: Colors.red,
+                          backgroundCursorColor: Colors.black)
+                  ),
+                  Container(
+                      height:25.0,
+                      child: TextField(
+                          key: const ValueKey<String>('text4'),
+                          controller: TextEditingController(text: 'Hello4'))
```

Comment:
Usually we put the closing parenthesis on a newline if the opening one is on its own line, though I can't find a link to that in the style guide at the moment.
```dart
                      child: TextField(
                          key: const ValueKey<String>('text4'),
                          controller: TextEditingController(text: 'Hello4')
                      )
```